### DISCUSSION TITLE
Custom pos for dependency parsing

### DISCUSSION THREAD

**Discussion by saxtell-cb** [explosion/spaCy#9510] [Help: Coding & Implementations] [answered]
I'm wondering if it's possible to use custom pos tags during spaCy's dependency parsing. I have a number of custom pos tags that I want spaCy to use instead of its own in order to derive the dependency parse.

Thanks in advance.

**Comment by polm** [comment, marked as answer]
The dependency parser does not use POS tags. There are three attributes relevant to this discussion:

1. Dependency relations in the `.dep` attribute. These are values like `nmod`, `amod`, `cconj`, and come from Universal Dependencies in the pretrained models, but any value can be used.
2. POS tags in the `.pos` attribute. These come from Universal Dependencies, and spaCy enforces that (you can't make up new ones).
3. fine-grained, language-specific tags in the `.tag` attribute. These can be anything. 

You can customize the dependency relation values or the `.tag` attribute values. However the parser does not use POS tags.

Also, maybe you really do want custom POS tags, but usually it's much easier to use entity labels. If you haven't considered that maybe take a look at it.

**Reply by saxtell-cb** [reply]
Thanks @polm! I can see now that I should have phrased this differently. Your first sentence answers the question for me. We are actually using the `.tag` value to pass along our own pos tags, so all good there. Thanks again!
